src/openrct2/ride/RideRatings.h

```diff
@@ -54,6 +54,11 @@ struct RideRatingCalculationData
     uint16_t num_brakes;
     uint16_t num_reversers;
     uint16_t station_flags;
+    uint16_t cycle_proximity_x;
+    uint16_t cycle_proximity_y;
+    uint16_t cycle_proximity_z;
+    uint8_t cycle_proximity_track_type;
+    bool cycle_detection_toggle;
 };
```

Comment:
You have broken the SV6 format doing this. This is why we haven't tried doing this before.

Reply:
Ah. Would it work if I put these in a different struct?

Reply:
Yes, remember that it also needs to reset for everyone when a new player joins a multiplayer game too.